server/package.json

```diff
@@ -15,19 +15,27 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/identity": "^4.2.0",
+    "@azure/identity": "^4.4.1",
     "@azure/storage-blob": "^12.23.0",
     "@sendgrid/mail": "^8.1.3",
     "axios": "^1.7.2",
     "azure-storage": "^2.10.7",
     "body-parser": "^1.20.2",
     "cors": "^2.8.5",
     "dotenv": "^16.4.5",
+<<<<<<< Updated upstream
     "express-session": "^1.18.0",
     "google-auth-library": "^9.14.1",
     "express": "^4.21.0",
     "fluent-ffmpeg": "^2.1.3",
     "fs": "^0.0.1-security",
+=======
+    "express": "^4.18.2",
+    "express-session": "^1.18.0",
+    "fluent-ffmpeg": "^2.1.3",
+    "fs": "^0.0.1-security",
+    "google-auth-library": "^9.14.1",
+>>>>>>> Stashed changes
```

Comment:
Identify common packages, and use the latest version. Remove this >>> and === lines.

Reply:
After pushing the code I saw that and I have resolved it